src/App.tsx

```diff
@@ -1,16 +1,78 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+type State = {
+  counter: number,
+};
 
-  add100 = () => {};
+class App extends React.Component<{}, State> {
+  state = {
+    counter: 0,
+  };
 
-  increase = () => {};
+  addOne = () => {
+    this.setState((state) => ({
+      counter: state.counter + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState((state) => ({
+      counter: state.counter + 100,
+    }));
+  };
+
+  increase = () => {
+    this.setState((state) => ({
+      counter: state.counter + 1,
+    }));
+
+    if (this.state.counter % 5 === 0) {
+      this.setState((state) => ({
+        counter: state.counter + 100,
+      }));
```

Comment:
```suggestion
    addOne

    if (this.state.counter % 5 === 0) {
      add100
```
Для чого дублювати код, якщо в тебе вже є функції, які це роблять?